### PR TITLE
Makefile cleanups

### PR DESCRIPTION
**Pull request by thockin**
Cleaned up while debugging elsewhere.

### CODE REVIEW

File: build/root/Makefile.generated_files [hunk 1]
```diff
@@ -222,7 +222,7 @@ RUN_GEN_DEEPCOPY =                                                          \
                 --v $(KUBE_VERBOSE)                                         \
                 --logtostderr                                               \
                 -i $$(cat $(META_DIR)/$(DEEPCOPY_GEN).todo | paste -sd, -)  \
-                --bounding-dirs $(PRJ_SRC_PATH),,"k8s.io/api"               \
+                --bounding-dirs $(PRJ_SRC_PATH),"k8s.io/api"                \
```
Comment (ixdy) [inline build/root/Makefile.generated_files]: I'm guessing the `,,` was a typo @caesarxuchao ?

Comment (thockin) [inline build/root/Makefile.generated_files]: yep, gets interpreted as and empty list element

**Pull request merged**